### DISCUSSION TITLE
Bulk edit

### DISCUSSION THREAD

**Discussion by fjullien** [inventree/InvenTree#3918] [Q&A] [answered]
Lets say I have 1000 resistors in the database. I now want to add a new parameter to resistors, let's say ROHS status.
Is it possible to have a bulk edit mode in some way ?

Or let's say I have a maximum temperature parameter. People prefix the temperature with a "+" and other don't.
Can I have an overview of this parameter over all components in order to make the style consistent ?
Some kind of spreadsheet style editor would be cool.

**Comment by matmair** [comment, marked as answer]
@fjullien these are two questions and suggestion so I will enumerate them for clearer discussion:
a) There is no bulk edit mode currently - I would suggest using the [python module](https://docs.inventree.org/en/latest/api/python/python/) or [API](https://docs.inventree.org/en/latest/api/api/)

b) Not on a global level but from the first category downwards. Example:
https://demo.inventree.org/part/category/2/#
![grafik](https://user-images.githubusercontent.com/66015116/201215703-f1ae0d6e-ea2b-4cef-a728-28e347df5d09.png)

c) A spreadsheet-style editor would be cool but a lot of work. If you know how to do that in JS feel free to submit a PR :-) The team is mostly electrical and civil engineers so only self-taugth frontend-knowledge

**Reply by fjullien** [reply]
Thanks for your answers.
I'm not a JS person at all, so no contribution for now :)
However, I didn't pay attention to the python module and plugins. It seems very interesting (also for custom BOM export format).
